alerting/src/main/kotlin/org/opensearch/alerting/settings/AlertingSettings.kt

```diff
@@ -156,5 +173,94 @@ class AlertingSettings {
             -1L,
             Setting.Property.NodeScope, Setting.Property.Dynamic
         )
+
+        val TOTAL_MAX_ACTIONS_PER_TRIGGERS = Setting.intSetting(
+            "plugins.alerting.max_actions_across_triggers",
+            DEFAULT_TOTAL_MAX_ACTIONS_PER_TRIGGERS,
+            -1, MaxActionsTriggersValidator(internalClient),
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
+        val TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS = Setting.intSetting(
+            "plugins.alerting.total_max_actions_across_triggers",
+            DEFAULT_TOTAL_MAX_ACTIONS_ACROSS_TRIGGERS,
+            -1, TotalMaxActionsTriggersValidator(internalClient),
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
+
+        internal class TotalMaxActionsTriggersValidator(val client: Client?) : Setting.Validator<Int> {
```

Comment:
Rename function to make it more clear like `TotalMaxActionsAcrossTriggersValidator`